rvi_capture.py

```diff
@@ -223,9 +223,9 @@ def _chunk_to_packet(self, chunk):
         [pid] = SL32.unpack(UB32.pack(pid))
         [epid] = SL32.unpack(UB32.pack(epid))
         [svc_class] = SL32.unpack(UB32.pack(svc_class))
-        iface_name = iface_name.rstrip(b'\0').decode('ascii') + str(if_unit)
-        comm = comm.rstrip(b'\0').decode('ascii')
-        ecomm = ecomm.rstrip(b'\0').decode('ascii')
+        iface_name = iface_name.rstrip(b'\0').decode('utf-8') + str(if_unit)
```

Comment:
Could you please keep `iface_name` as `ascii`? I don't think it should ever be not ascii, and we would like to know in that case.

Reply:
I test. You're right